### PR TITLE
Updated testing

### PR DESCRIPTION
**Pull request by threeaccents**

### CODE REVIEW

File: fsutil_test.go [hunk 1]
```diff
@@ -84,13 +110,18 @@ func TestCopyDirectory(t *testing.T) {
 }
 
 func TestIsEmpty(t *testing.T) {
+	dir, err := ioutil.TempDir("", "test")
+	if err != nil {
+		t.Error("Error creating temp file", err)
+	}
+
```
Comment (jesselucas) [inline fsutil_test.go]: We should add `defer os.RemoveAll(dir)` to clean up the temp dir.

Comment (rodzzlessa24) [inline fsutil_test.go]: @jesselucas  pushed up the changes.

File: fsutil_test.go [hunk 2]
```diff
@@ -106,6 +137,9 @@ func TestIsEmpty(t *testing.T) {
 			t.Errorf("Expected %t, Result was: %t", test.expected, result)
 		}
 	}
+
+	// Remove temp dir
+	os.RemoveAll(dir)
```
Comment (jesselucas) [inline fsutil_test.go]: It's better practice to use defer incase something else returns out of the function before the `RemoveAll` is called.

Comment (rodzzlessa24) [inline fsutil_test.go]: I changed it. I thought that `t.Error` didn't actually return the function it was more like a print of the error that's why I hadn't used defer

**Pull request merged**